src/react/forms.jsx

```diff
@@ -5,9 +5,28 @@
 
 'use strict';
 
+var formControlClass = 'mui-form-control',
+    formGroupClass = 'mui-form-group',
+    floatingLabelBaseClass = 'mui-form-floating-label',
+    floatingLabelActiveClass = floatingLabelBaseClass + '-active',
+    animationName = 'mui-form-floating-label-inserted',
+    _supportsPointerEvents;
 
-var jqLite = require('../js/lib/jqLite.js'),
-    forms = require('../js/forms.js');
+var React = require('react');
```

Comment:
That would be ideal but it's tricky to do it with Browserify since it will inline the source even if the require() statement is in a conditional. We can also leave it to the developer to bundle React and mui-react.js using their own build script.